docs/components/table.md

```diff
@@ -66,7 +66,7 @@ export default {
 ```
 ## Props
 ### Hover
-highlight the table row on hover
+Highlight the table row on hover
```

Comment:
now that highlight is true by default, lets change this example and doc to explain its always on and you can set it to false `hasHover="false"`